### ISSUE TITLE
lstchain_data_muon_analysis script broken

### ISSUE THREAD

**Issue by rlopezcoto** [cta-observatory/cta-lstchain#330]
`lstchain_data_muon_analysis.py` was performing the muon analysis from `R0` data level and `lstchain_data_muon_analysis_dl1.py` was performing it starting at `DL1` data level. After #304 the muon data analysis has already been inserted in the `dl0_to_dl1` script. 
`lstchain_data_muon_analysis.py` has fallen pretty much behind, so not to have to maintain two scripts, I propose to eliminate it. @moralejo, @FrancaCassol is that ok?

**Comment by moralejo** [comment]
Ok with me.

**Comment by FrancaCassol** [comment]
Ho @rlopezcoto , @moralejo,
I agree if it possible to run dl0_to_dl1 only  for selecting muon events.

Actually, it would have been wiser to define a muon searcher function  that could have been called in the dl0_to_dl1.py (by the way we should change the name since it starts from r0) and in this script so to have only one muon code to maintain. 

I see also that in [https://github.com/cta-observatory/cta-lstchain/pull/304](https://github.com/cta-observatory/cta-lstchain/pull/304) @moralejo  kept the unnecessary treatment of bad_pixels, as already commented,  this is already done in the calibrator now.

**Comment by rlopezcoto** [comment]
> this is already done in the calibrator now.

do you mean after #307?

**Comment by moralejo** [comment]
> I agree if it possible to run dl0_to_dl1 only for selecting muon events.
> 
> Actually, it would have been wiser to define a muon searcher function that could have been called in the dl0_to_dl1.py (by the way we should change the name since it starts from r0) and in this script so to have only one muon code to maintain.
> 
Hi, the ring analysis is the same, and the pre-selection of images which may be rings is also the same (a very simple check of number of pixels with l large signals). So there is no big deal of repetition anywhere. In the r0_to_dl1 case one does the re-calibration of the image, but that is something you cannot do if you start from dl1.

> I see also that in #304 @moralejo kept the unnecessary treatment of bad_pixels, as already commented, this is already done in the calibrator now.
Yes, it is redundant, but it allows me to work with
